### PR TITLE
Blacked all code and added pytest

### PR DESCRIPTION
**Pull request by trym-b**
Blacked non blacked files

Enabled pytest with simple dummy test. Had to disable a pylint test since it acted up when used with zivid module

Added requirements.txt file for ci, all platforms need to have pip available

### CODE REVIEW

File: .pylintrc [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [MESSAGES CONTROL]
-disable=missing-docstring
+disable=missing-docstring,
+        F0001, # disabled because pylint tries to lint some internal zivid setup files when used with pytest
```
Comment (nedrebo) [inline .pylintrc]: Which test is this: Can you add a human readable name?
What is "internal zivid setup files"?

File: continuous-integration/setup.sh [hunk 2]
```diff
@@ -28,4 +28,6 @@ else
     exit 1
 fi
 
+pip install -r requirements.txt || exit $?
```
Comment (nedrebo) [inline continuous-integration/setup.sh]: Relative path will only work in if your WD is the continuous-integration dir.

File: continuous-integration/test.sh [hunk 3]
```diff
@@ -11,4 +11,7 @@ pip install $ROOT_DIR || exit $?
 
 python -c "import zivid" || exit $?
 
+pip install pytest || exit $?
+python -m pytest || exit $?
```
Comment (nedrebo) [inline continuous-integration/test.sh]: Why don't you use the requirements file?

File: continuous-integration/test/test_dummy.py [hunk 4]
```diff
@@ -0,0 +1,3 @@
+def test_dummy():
```
Comment (nedrebo) [inline continuous-integration/test/test_dummy.py]: Why not a proper test like test_import_zivid?

Comment (trym-b) [reply continuous-integration/test/test_dummy.py]: Trying to keep this contained. This PR is purely modifying infrastructure.

File: .pylintrc (hunk 1 above)
Comment (trym-b) [reply .pylintrc]: It keeps failing when trying to lint setup py through the pytest module. No module named "/host_dir/setup.py"

**Comment by trym-b** [comment]
Ready for next round

File: continuous-integration/test.sh (hunk 3 above)
Comment (trym-b) [reply continuous-integration/test.sh]: Good point: fixed

File: continuous-integration/setup.sh (hunk 2 above)
Comment (trym-b) [reply continuous-integration/setup.sh]: Removed completely, was redundant

File: .pylintrc [hunk 5]
```diff
@@ -1,5 +1,6 @@
 [MESSAGES CONTROL]
-disable=missing-docstring
+disable=missing-docstring,
+        F0001, # No module named ... - disabled because pylint tries to lint ROOT_DIR/setup.py
```
Comment (nedrebo) [inline .pylintrc]: Why is this suddenly required, it did work before. I cannot see any change in PR that should affect this.

Comment (trym-b) [reply .pylintrc]: Because of the following commits in the next branch waiting for this. I can take it out.

**Comment by trym-b** [comment]
https://travis-ci.org/nedrebo/upgraded-funicular/jobs/541234208
We even get the problem for the dummy test

**Comment by trym-b** [comment]
Ready for next round

File: continuous-integration/test.sh [hunk 6]
```diff
@@ -11,4 +11,8 @@ pip install $ROOT_DIR || exit $?
 
 python -c "import zivid" || exit $?
 
+pip install -r "$SCRIPT_DIR/requirements.txt" || exit $?
+
+python -m pytest || exit $?
```
Comment (nedrebo) [inline continuous-integration/test.sh]: Doesn't this need a dir as argument else I guess you get whatever is WD.

File: .pylintrc (hunk 5 above)
Comment (nedrebo) [reply .pylintrc]: Please do

**Pull request merged**